content/datum.ttl

```diff
@@ -3,8 +3,11 @@
 :datum
     skos:prefLabel "datum"@en;
     skos:definition '''
-      Optional data that can be associated to a utxo when it is created.
+      Optional data that can be associated to a UTXO when it is
```

Comment:
Are you manually re lining this, or have you integrated a linter? changes like this should either be enforced (with assistance) or ignored. They introduce noise in the git history.

Reply:
+1

This is a general practice that does improve code-consistency, but also removes friction in code-review: https://medium.com/@schrockn/on-code-reviews-b1c7c94d868c#1603 

Nb. per a previous comment, we should also identify other points of friction in this code-base to remove the amount of manual hand turning we have to do, e.g. like isort https://pycqa.github.io/isort/ but for predicate ordering.